protobuf/module_msg.proto

```diff
@@ -450,12 +450,17 @@ message BufferArg {
 }
 
 /**
- * The Bypass module forwards packets without any processing. It requires no parameters to initialize. Bypass is useful primarily for testing and performance evaluation.
+ * The Bypass module forwards packets by emulating pre-defined packet processing overhead.
+ * It burns cpu cycles per_batch, per_packet, and per-bytes.
+ * Bypass is useful primarily for testing and performance evaluation.
  *
  * __Input Gates__: 1
  * __Output Gates__: 1
  */
 message BypassArg {
+  uint32 cycles_per_batch = 1;
+  uint32 cycles_per_packet = 2;
+  double cycles_per_byte = 3;
```

Comment:
Why `double` here instead of `uint32` like the other members?

Reply:
Just following the suggestions from the issue discussion. uint32 also fine.